docs/_static/json-example-format.js

```diff
@@ -0,0 +1,36 @@
+   $( document ).ready(function() {
```

Comment:
It is the code used in the standard for displaying collapsible excerpts from JSON files (https://github.com/open-contracting/standard/blob/26075ede08503e3e10e097af20b40cc85107d4cc/standard/docs/en/_static/json-example-format.js)

Reply:
Can you add a comment at the top of the file:

`See https://github.com/open-contracting/standard/blob/1.1-dev/standard/docs/en/_static/json-example-format.js`

This will help ensure that if we change the code here, we remember to change it in the standard.

Reply:
Added in https://github.com/open-contracting/infrastructure/pull/72/commits/c93a5b9bcfc04da9a405f4b2569f66d5ed3b9752